Dockerfile

```diff
@@ -2,11 +2,10 @@ ARG GDAL_VERSION
 FROM remotepixel/amazonlinux:gdal${GDAL_VERSION}-py3.7
 
 ARG GDAL_VERSION
+ARG REQ_VERSION
 ENV GDAL_VERSION $GDAL_VERSION
 
-RUN pip install cython==0.28
-
-COPY requirements-gdal${GDAL_VERSION}.txt requirements.txt
+COPY requirements-${REQ_VERSION}.txt requirements.txt
```

Comment:
Now that we know the underlying root issue was inversion of 2.4 lat/lon parameters to lon/lat in 3.0, why do we need the extra complexity to handle both versions? Why not just commit to 3.0

Reply:
That is fine with me. But when you wrote to abandon the PR, what do you mean? Make the change in some other branch? (In this repo?)

Reply:
I now understand that the original cogeo repo supports both 2.4 and 2.5. This PR removes changes WE made to simplify merging future changes made to the original cogeo repo.

Reply:
2.4 and 3.0. I could make a second docker file, and a second makefile just for our changes... That would really make it easier to incorporate upstream changes.